Task 3/CompressionAlgorithm

```diff
@@ -0,0 +1,57 @@
+package dev.benedikt.compression.sixpack
```

Comment:
Make a folder and follow the file structure as mentioned in Readme. Rename the file with .kt extension

Reply:
folder structure followed as per your instructions again pull request  created.Hence closing the PR